helix-core/src/main/java/org/apache/helix/manager/zk/CallbackHandler.java

```diff
@@ -110,9 +112,12 @@
    * define the next possible notification types
    */
   private static Map<Type, List<Type>> nextNotificationType = new HashMap<>();
+
   static {
-    nextNotificationType.put(Type.INIT, Arrays.asList(Type.CALLBACK, Type.FINALIZE));
-    nextNotificationType.put(Type.CALLBACK, Arrays.asList(Type.CALLBACK, Type.FINALIZE));
+    nextNotificationType
```

Comment:
Is this agreed? I am not sure. I was preferring to have a separate type to by pass even if there is a callback wrong order issue.

Reply:
Let's just use CALLBACK for the periodic triggering.